text/fuzz/corpus/from_test_parse_1

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
Wondering, is it intentional that all these corpus files start with a single empty line?